static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -131,13 +148,38 @@ function DefaultGroupEventDetailsContent({
         />
       </StyledDataSection>
       {event.userReport && (
-        <EventDataSection title={t('User Feedback')} type="user-feedback">
-          <EventUserFeedback
-            report={event.userReport}
-            orgSlug={organization.slug}
-            issueId={group.id}
-            showEventLink={false}
-          />
+        <EventDataSection
+          title={t('User Feedback')}
+          type="user-feedback"
+          actions={
+            // Button is disabled if promptLoading or promptError.
+            <ErrorBoundary mini>
+              <Button
+                size="xs"
+                onClick={isPromptDismissed ? showPrompt : dismissPrompt}
+                title={
+                  isPromptDismissed
+                    ? t('Unhide feedback on all issue details')
+                    : t('Hide feedback on all issue details')
+                }
+                disabled={promptError || promptLoading || isPromptDismissed === undefined}
+              >
```

Comment:
what does it mean if `isPromptDismissed` undefined?

Reply:
https://github.com/getsentry/sentry/blob/138a24e16e507fdb66aa231a0f4300bcb37d62ce/static/app/actionCreators/prompts.tsx#L162 I think some kind of error with the backend. I'm assuming when this occurs promptError=true.

Reply:
i think it shouldn't be disabled if `isPromptDismissed === undefined` -- it looks like it's undefined anytime the prompt is not dismissed, even with no error


<img width="931" alt="SCR-20240715-lqtw" src="https://github.com/user-attachments/assets/e58518eb-3c64-4d99-9c3b-0e4ae36d2042">

Reply:
oh interesting.. thanks for testing this case!

Reply:
actually, hold on... `isPromptDismissed` is supposed to return `true` or `false` so the logic isn't adding up. maybe it's a vercel preview thing. does https://sentry-sdks.sentry.io/issues/5291936195/events/e24fd30c0f2b47a9b20bc5014de8e493/?project=-1&referrer=event-or-group-header&statsPeriod=90d work as expected for you?

Reply:
Using the latest commit, the button is disabled for me

Reply:
Agreed on slack to show the feedback if error or `isPromptDismissed === undefined`, use `LoadingIndicator` when loading